src/modules/CanvasResize/info.json

```diff
@@ -15,12 +15,12 @@
         "x": {
             "type": "integer",
             "desc": "X-cord of the top left corner of the image on the canvas",
-            "default": 500
+            "default": 200
         },
         "y": {
             "type": "float",
             "desc": "Y-cord of the top left corner of the image on the canvas",
-            "default": 500
+            "default": 200
```

Comment:
these unrelated changes are still there. Are they required? If not, it is best to remove them imo.

Reply:
> I don't remember exatcly but maybe it's default value was changed because of this bug or something so I changed it to the original after the bug was fixed(most probably, I don't remeber exactly why I changed this but it was something like this)

Commented on the earlier review
New Values:
![Screenshot from 2020-07-12 23-42-16](https://user-images.githubusercontent.com/42492389/87253505-6d3d5d80-c499-11ea-8a3b-e8c485a17523.png)
Old Values:
![Screenshot from 2020-07-12 23-42-35](https://user-images.githubusercontent.com/42492389/87253512-77f7f280-c499-11ea-8291-8a4c6fb8ee0b.png)

Reply:
Does it matter though? If not, it will be easier to review less than 32 changes :sweat_smile: